bin/mail/2016-08-24-updated-terms/send-real-mails

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+
+# 367743 total teachers
+
+../send-to-mailing-list --limit 368000 8-22-updated-terms ./teachers.csv
```

Comment:
Please increase the limit, this is sufficiently important and teacher signups are accelerating.